mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PeliasCallbackHandler.java

```diff
@@ -28,12 +26,17 @@
    * Creates a {@link Place} from a {@link Feature} with a name that matches the given title and
    * notifies the listener that a place has been successfully retrieved.
    * @param title
-   * @param features
+   * @param response
    * @param listener
    */
-  public void handleSuccess(String title, List<Feature> features,
+  void handleSuccess(String title, Response<Result> response,
```

Comment:
It should be fine. This is an internal callback only. We try to make everything in the `com.mapzen.places.api.internal` package protected so it will not show up in the public API.